<launch>
  <node name="hand_tracker" pkg="cs7633-project" type="hand_tracker.py" output="screen"/>
</launch>